provisioning/src/handlers/portfolioDrafts/post.ts

```diff
@@ -0,0 +1,31 @@
+import { Context, HttpRequest } from '@azure/functions'
+const { CosmosClient } = require("@azure/cosmos");
```

Comment:
Is there a reason for using require over import?

Reply:
I like the suggested ES6 syntax.  Makes it similar to the line above.

Reply:
Nope, probably just copypasta that we should fix.